ddpui/ddpdbt/dbt_service.py

```
@@ -253,3 +274,329 @@ def check_repo_exists(gitrepo_url: str, gitrepo_access_token: str | None) -> boo
+    try:
+        profile = prefect_service.get_dbt_cli_profile_block(dbt_cli_profile_block.block_name)[
+            "profile"
+        ]
+        profile_dirname = Path(dbt_project_params.project_dir) / "profiles"
+        os.makedirs(profile_dirname, exist_ok=True)
+        profile_filename = profile_dirname / "profiles.yml"
+        logger.info("writing dbt profile to " + str(profile_filename))
+        with open(profile_filename, "w", encoding="utf-8") as f:
+            yaml.safe_dump(profile, f)
+    except Exception as err:
+        logger.error(f"failed to write profiles.yml: {str(err)}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)